### ISSUE TITLE
make sure the DAQ_28Jan22 area is merged into next release

### ISSUE THREAD

**Issue by wesketchum** [SBNSoftware/sbndaq#36]
...or that the developments we want there are done.

**Comment by gennadiy-fnal** [comment]
@jzettle @ascarpel please commit and push your code changes.

**Comment by jzettle** [comment]
@gennadiy-fnal I have done this already. Everything I have updated is pushed to this branch https://github.com/SBNSoftware/sbndaq-artdaq-core/tree/feature/jzettle_triggerupdates

**Comment by gennadiy-fnal** [comment]
@jzettle  What about this ?
$ cd sbndaq_artdaq_core
15:33:03icarus@icarus-evb06:~/DAQ_DevAreas/DAQ_28Jan2022/srcs/sbndaq_artdaq_core feature/jzettle_triggerupdates*
$ git diff
diff --git a/sbndaq-artdaq-core/Overlays/ICARUS/ICARUSTriggerUDPFragment.hh b/sbndaq-artdaq-core/Overlays/ICARUS/ICARUSTriggerUDPFragment.hh
index 987877d..c4c33f1 100644
--- a/sbndaq-artdaq-core/Overlays/ICARUS/ICARUSTriggerUDPFragment.hh
+++ b/sbndaq-artdaq-core/Overlays/ICARUS/ICARUSTriggerUDPFragment.hh
@@ -2,6 +2,7 @@
 #define sbndaq_artdaq_core_Overlays_ICARUS_ICARUSTriggerUDPFragment_hh

 #include "artdaq-core/Data/Fragment.hh"
+#include "sbndaq-artdaq-core/Trace/trace_defines.h"
 #include "cetlib_except/exception.h"

 #include <iostream>
@@ -119,6 +120,7 @@ icarus::ICARUSTriggerInfo icarus::parse_ICARUSTriggerString(const char* buffer)
   //std::string trig_name = sections[0];
   ICARUSTriggerInfo info;
   //this should be agnostic to any length (and order) string that follows the key, value, format unless another timestamp is added
+  /*
   for(unsigned int i = 0; i < sections.size(); i = i + 2)
   {
     //large or statement should do this just fine, need to separate out TS stuff
@@ -209,9 +211,10 @@ icarus::ICARUSTriggerInfo icarus::parse_ICARUSTriggerString(const char* buffer)
       info.cryo2_west_counts = std::stol(it->second);

   }
+  */
   //old implementation preserved in case of emergency, rigid hardcoded version where string must appear in a specific order
-  /*
-  info.name = sections[0];
+
+  info.name = sections[2];
   //t->setHardwareTS_Type(sections[0]);
   info.event_no = std::stol(sections[3]);
   info.seconds = std::stoi(sections[4]);
@@ -232,7 +235,7 @@ icarus::ICARUSTriggerInfo icarus::parse_ICARUSTriggerString(const char* buffer)
       info.beam_nanoseconds = std::stol(sections[31]);

     }
-  */
+
   return info;
 }

**Comment by jzettle** [comment]
I will need to look at it. Whatever is in here now it needs to be updated to what is in that branch, that is what I have mostly tested explicitly. I should have included any divergences within that area. I think the issue is initially this file was changed in this running area without being pushed back to my branch afterwards so when I edited recently I did not know the file was different in this area. I brought in any additions I could find when I made my initial tests.
